java/pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.simpledao</groupId>
     <artifactId>simpledao</artifactId>
-    <version>2.9.8</version>
+    <version>2.9.10</version>

```

Comment:
I believe the github action [pipeline ](https://github.com/utahdws/webteam-simpledao-fork/actions/workflows/pr_open.yml)should auto increment this for us.  I'm not sure why the git hub package has 2.9.9 [https://github.com/utahdws/webteam-simpledao-fork/packages/964031](url) but the develop branch is showing 2.9.8.  Maybe there is an issue with the pipeline, not sure.